mlrun/feature_store/api.py

```diff
@@ -230,6 +230,11 @@ def _get_offline_features(
             "entity_timestamp_column param "
             "can not be specified without entity_rows param"
         )
+    if isinstance(target, BaseStoreTarget) and not target.support_pandas:
```

Comment:
`isinstance(target, BaseStoreTarget)` as opposed to?

Reply:
the hint in the target parameter is DataTargetBase.

DataTargetBase does not have support_pandas:)